opm/autodiff/FlowMainEbos.hpp

```diff
@@ -52,6 +53,28 @@
 
 namespace Opm
 {
+
+    /// \brief Gather cell data to global random access iterator
+    /// \tparam ConstIter The type of constant iterator.
+    /// \tparam Iter The type of the mutable iterator.
+    /// \param grid The distributed CpGrid where loadbalance has been run.
+    /// \param local The local container from which the data should be sent.
+    /// \param global The global container to gather to.
+    /// \warning The global container has to have the correct size!
+    template<class ConstIter, class Iter>
+    void gatherCellDataToGlobalIterator(const Dune::CpGrid& grid,
```

Comment:
I'd prefer this not to be in the `Opm` namespace. (or, if it is usable in a more general context, it should not be part of this file.) Make it a static member of the FlowMainEbos class!?